addons/hr_skills/views/hr_employee_cv_templates.xml

```diff
@@ -85,19 +86,20 @@
         </t>
 
         <t t-set="has_languages" t-value="skill_type_language and any(skill_line.skill_type_id == skill_type_language for skill_line in o.employee_skill_ids)"/>
-        <div class="o_main_panel mt-4" t-if="has_languages">
+        <div class="o_main_panel w-100 align-top mt-4 px-3 pt-1 pb-0" t-if="has_languages">
```

Comment:
`align-top` is related to text, so it should be at the end (same for the others below).